spec/requests/settings_request_spec.rb

```diff
@@ -15,15 +15,15 @@
     describe '`settting` params' do
       let(:params) { { setting: { project_name: 'Shiny new project' } } }
 
-      it { should change { Setting.project_name }.to('Shiny new project') }
+      it { will change { Setting.project_name }.to('Shiny new project') }
```

Comment:
Hmm, in this case, I don’t really see how "It will change…" is easier to understand than "It should change…"?

Reply:
That fixes a deprecation warning in newer versions of rspec